book/src/feature/item_type.md

```diff
@@ -0,0 +1,24 @@
+# Item Type
+
+There are some example supported:
+```
+pub type Id = u64;
+
+pub fn handle_type_alias_id(input: Id) -> Id {
+    input
+}
+pub struct TestModel {
+    pub id: Id,
+    pub name: String,
+}
+
+pub fn handle_type_alias_model(input: Id) -> TestModel {
+    TestModel {
+        id: input,
+        name: "TestModel".to_owned(),
+    }
+}
+```
+
+You could use raw `ItemType` Id, or use It in Model.
```

Comment:
uber-nit: would be great to at least use `Grammarly` to auto check grammar of doc

Reply:
You can use the Id, or you can use the Id in the Model.
But ItemType in Generic is not supported yet.(like `SyncReturn<Id>`)

Thank you. I started using the free